recipe/migrations/arch_rebuild.txt

```diff
@@ -691,3 +691,6 @@ lammps
 phonopy
 pyscal
 pygplates
+libcublas
+libcublas-dev
+libcublas-static
```

Comment:
FWIW we only need the feedstock name. All outputs of the feedstock will be handled the same with the migration PR

Also generally it helps to add items to the middle of the list somewhere to avoid conflicts

Dropped `-dev` & `-static` in PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4319 ). Also moved `libcublas` near other `libcu*`s